### ISSUE TITLE
Link to /research page from the blog

### ISSUE THREAD

**Issue by jywarren** [publiclab/plots2#989]
Hi, this is a `first-timers-only` issue. This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

If you have contributed before, consider leaving this one for someone new, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

## The problem

We have a great blog at: https://publiclab.org/blog

But people who read it there don't necessarily find their way into our https://publiclab.org/tools page or the https://publiclab.org/research page, which would be great.

## Solution

We should add a new pair of links below the "Blog" header to go to these two.

The code for the blog subheader is here: https://github.com/publiclab/plots2/blob/master/app/views/tag/blog.html.erb#L10

```ruby
      <p><a href="/wiki/blog"><%= t('tag.blog.about_the_blog') %></a></p>
```

We should add two more links, to https://publiclab.org/tools (Text: `Methods` -- we're changing the name of this soon, so we'll just use that) and https://publiclab.org/research (Text: `Latest updates`), in the same `<p>` tag. 

The part with `<%= t('tag.blog.about_the_blog') %>` is referring to our translation files -- so we can show these links in various language. The translations themselves have unique ids, and `tag.blog.about_the_blog` refers to this line in English: 

https://github.com/publiclab/plots2/blob/master/config/locales/views/tag/blog/en.yml#L9

To add the new text links in a way that will be translatable, you can add new lines to that file with new unique ids, like:

```yml
      research: "Latest updates"
      methods: "Methods"
```

The template code would then be:  `<%= t('tag.blog.research') %>` -- etc.

## Steps to Fix

- [ ] claim this issue with a comment here, below, and ask any clarifying questions you need
- [ ] set up a repository locally following the README instructions, and make sure that all tests pass
- [ ] try to fix the issue following the steps above, but even before you're done, you can:
- [ ] commit your changes and start a pull request (see [contributing to Public Lab software](https://publiclab.org/wiki/contributing-to-public-lab-software)) but mark it as "in progress" if you have questions or if you haven't finished
- [ ] alert someone via the developers list (see below) to have your pull request merged. This may take a few extra steps depending on when you do it, but we'll help you out!

Please email the developers list (see https://publiclab.org/developers) if you have questions!

**Comment by rodomvp** [comment]
I'm going to take a shot at this - first time contributing to open source so I'll do my best to solo it but I'll ask any clarifying questions that I need.

**Comment by jywarren** [comment]
Super! Many thanks!

**Comment by jywarren** [comment]
Closed in #993!
